webapp/src/Controller/PublicController.php

```diff
@@ -79,6 +82,18 @@ public function scoreboardAction(
 
         if ($static) {
             $data['hide_menu'] = true;
+            $submissions = $this->submissionService->getSubmissionList(
+                [$contest->getCid() => $contest],
+                new SubmissionRestriction(valid: true),
+                paginated: false
```

Comment:
For long running contests we would still get a long list?

Reply:
For one team? We would if the team submits a lot. We can add this behind an option if this becomes a problem.

Reply:
is the modal scrollable?

Reply:
It is not, it's sorted by submission time in decreasing order.

Reply:
Note that this now uses AJAX to load the JSON data and thus is less data to transmit. We can add pagination if we want.

You can scroll in the modal btw, not sure why I said you couldn't.